### ISSUE TITLE
ONIX audiobook test not matching the right code number

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#670]
The current xpath just looks for the value 81:

> LET audiobook be the result of calling [check for node](https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/onix-metadata/#check-for-node) on onix, /ONIXMessage/Product/DescriptiveDetail[PrimaryContentType = "81" or ProductContentType = "81"]

<strike>But 01/81 indicates an audiobook while 09/81 is the code for wcag 2.1, so this triggers false positives in non-audiobooks. `ProductFormFeatureType = "01"` should also be specified.</strike>

See https://github.com/w3c/publ-a11y/issues/670#issuecomment-2706817345 for the possible issue here.

**Comment by chrisONIX** [comment]
The current xpath is correct except for the value 81 - xx should be a value from list 81 -  https://ns.editeur.org/onix/en/81 not the value 81 itself.
/ONIXMessage/Product/DescriptiveDetail[PrimaryContentType = "xx" or ProductContentType = "xx"]
so the audiobook path would look like this 
/ONIXMessage/Product/DescriptiveDetail[PrimaryContentType = "01" or ProductContentType = "01"]

ProductFormFeatureType = "01" - this would indicate the color of the cover - ProductFormFeatureTypes are values from list 79 https://ns.editeur.org/onix/en/79 - the 09 code from list indicates that the other code will be a value from list 196

**Comment by mattgarrish** [comment]
Thanks, yes, I thought the 81 value was the problem causing prerecorded audio to display in a file I was running through the viewer code, but I realize that wasn't it now. I'm still trying to figure out where the bug is... 😦

**Comment by mattgarrish** [comment]
Well, at least I accidentally stumbled on something that needs fixing!

I figured out my problem, which seems to be that the contains function will match the lack of a child element to a string. I had changed the xpath I mentioned in #586 to a 1.0-compatible version and it was returning true because there wasn't a ProductFormType element in the test file. Didn't see that one coming, but this looks like the only place it could happen and I've reverted the xpaths...

**Comment by chrisONIX** [comment]
LET audiobook be the result of calling [check for node](https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/onix-metadata/index.html#check-for-node) on onix, /ONIXMessage/Product/DescriptiveDetail[PrimaryContentType = "81" or ProductContentType = "81"] .

change 81 to 01
